toml/src/main/kotlin/org/rust/toml/intentions/ExpandDependencySpecificationIntention.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.toml.intentions
+
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import com.intellij.psi.util.parentOfType
+import org.rust.cargo.CargoConstants
+import org.rust.lang.core.psi.ext.endOffset
+import org.rust.toml.isDependencyListHeader
+import org.toml.lang.psi.TomlKeyValue
+import org.toml.lang.psi.TomlLiteral
+import org.toml.lang.psi.TomlPsiFactory
+import org.toml.lang.psi.TomlTable
+import org.toml.lang.psi.ext.TomlLiteralKind
+import org.toml.lang.psi.ext.kind
+
+class ExpandDependencySpecificationIntention : TomlElementBaseIntentionAction<TomlKeyValue>() {
```

Comment:
Consider creating an opposite intention, i.e. to convert dependency table only with version to key-value form.
Of course, in separate PR